contracts/contracts/TACoApplication.sol

```diff
@@ -143,10 +144,13 @@ contract TACoApplication is IApplication, ITACoChildToRoot, OwnableUpgradeable {
         uint96 deauthorizing; // TODO real usage only in getActiveStakingProviders, maybe remove?
         uint64 endDeauthorization;
         uint96 tReward;
-        uint96 rewardPerTokenPaid;
+        uint160 rewardPerTokenPaid;
         uint64 endCommitment;
     }
 
+    uint256 public constant REWARD_PER_TOKEN_MULTIPLIER = 10 ** 4;
```

Comment:
What does this constant represent? What's the multiplier? Is this something for precision?

Reply:
yeah, name is not perfect. Thing is we have variable called `rewardPerToken`. With increase of precision now it's actually not per token but per 10000 tokens (that multiplier).

Reply:
`REWARD_CALCULATION_MULTIPLIER`?

Reply:
hmm? this name suits better for `FLOATING_POINT_DIVISOR`